mintpy/utils/utils1.py

```diff
@@ -156,7 +156,7 @@ def nonzero_mask(File, out_file='maskConnComp.h5', datasetName=None):
 
 
 def spatial_average(File, datasetName='coherence', maskFile=None, box=None,
-                    saveList=False, checkAoi=True):
+                    saveList=False, checkAoi=True, invertMask=False, threshold=None):
```

Comment:
I would recommend the following to be less ambiguous:
`invertMask` -> `reverseMask`
`threshold` -> `minValue`